Packs/Venafi/Integrations/VenafiV2/VenafiV2_test.py

```diff
@@ -0,0 +1,182 @@
+import json
+from freezegun import freeze_time
+
+MOCK_BASEURL = "https://test.com"
```

Comment:
Instead of mocking Client in each test, we can use pytest.fixture to craete the client before each test. See AMPv2_test.py for reference.